crawl.py

```diff
@@ -237,7 +237,7 @@ def crawl(self):
             try:
                 select_df.to_excel(self.args.save)
             except:
-                print("Bad Filename!!!")
+                print("Bad Filename!!!\nWarning:You probably  did not install openpyxl first, type \"pip install openpyxl\" to install the package ")
```

Comment:
Here is a redundant space between "probably" and "did" :)

Reply:
I fixed it in the latest version